src/appkit.cs

```diff
@@ -6523,6 +6538,12 @@ partial interface NSFont : NSSecureCoding, NSCopying {
 		[Internal]
 		[Export ("getAdvancements:forCGGlyphs:count:")]
 		void _GetAdvancements (IntPtr advancements, IntPtr glyphs, nuint glyphCount);
+
+		[Mac (10,15)]
+		[Static]
+		[Export ("monospacedSystemFontOfSize:weight:")]
+		[Internal]
+		IntPtr _MonospacedSystemFont (nfloat fontSize, double weight);
```

Comment:
```suggestion
		IntPtr _MonospacedSystemFont (nfloat fontSize, nfloat weight);
```

Reply:
-> `typedef CGFloat NSFontWeight;` and `CGFloat` is `nfloat`